tests/unit/test_charm_collect_status.py

```diff
@@ -202,6 +209,57 @@ def test_given_n3_route_is_missing_when_collect_unit_status_then_status_is_waiti
 
             assert state_out.unit_status == WaitingStatus("Waiting for the N3 route to be created")
 
+    @pytest.mark.parametrize(
+        "tac,plmns",
+        [
+            pytest.param(None, [PLMNConfig(mcc="001", mnc="01", sst=31)], id="tac_is_none"),
+            pytest.param(23, None, id="plmns_is_none"),
+            pytest.param(None, None, id="plmns_and_tac_are_none"),
+        ],
+    )
+    def test_given_fiveg_core_gnb_tac_and_plmns_are_none_when_collect_unit_status_then_status_is_waiting(  # noqa: E501
+        self, tac, plmns
```

Comment:
Need to add the fiveg_core_gnb relation.
Stay  blocked if the relation does not exits and waiting if the remote information is not there?